example/src/main/tut/typeclass/Monoid.md

```diff
@@ -5,8 +5,6 @@ title:  "Monoid"
 
 # Monoid
 
-**Category Theory**
```

Comment:
**Abstract Algebra**

Reply:
I removed all those references, maybe we can add the academic notes back later.